### ISSUE TITLE
Выполнить функцию

### ISSUE THREAD

**Issue by kozhin2012** [chayka/jQuery.Scroolly#13]
Можно ли не только css правила применить, но ещё запустить выполнение какой нибудь ф-ии?

**Comment by chayka** [comment]
да, можно, https://github.com/chayka/jQuery.Scroolly/wiki/Long-Story#rule-syntax

**Comment by kozhin2012** [comment]
Можно пример как например это заставить работать?
onCheckIn: function($element, rule){}
Не понятно что куда вписывать.

**Comment by AizSedaj** [comment]
`$('#header').scroolly([
            {
                from: 'doc-top',
                to: 'doc-top + 49el',
                css: {
                    position: 'absolute'
                },
                removeClass: 'scroll',
                onCheckIn: function($element, rule){
                    $("#scrolltotop").removeClass('show');
                }
            }
        ], $('body'));`
В скобках {} соответственно пишем код, который нужно выполнить.
